### PR TITLE
a better meteor example

### PR DESCRIPTION
**Pull request by TobyEalden**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -474,13 +482,26 @@ function getTrackerLoader(reactiveMapper) {
   };
 }
 
+function composeWithTracker(loadFunc) {
```
Comment (arunoda) [inline README.md]: I don't recommend using this.
Of course, you could use it.
But I don't really wanna put it on the README.

File: README.md [hunk 2]
```diff
@@ -457,13 +457,21 @@ const Container = compose(getReduxLoader(myMapper))(UIComponent)
...
   return (props, onData, env) => {
+  
     let trackerCleanup = () => null;
+  
     const handler = Tracker.nonreactive(() => {
       return Tracker.autorun(() => {
-				// assign the custom clean-up function.
-        trackerCleanup = reactiveMapper(props, onData, env);
+        // Store clean-up function if provided.
+        trackerCleanup = loaderFunc(props, onData, env) || (() => null);
```
Comment (arunoda) [inline README.md]: Yep. This is great.

**Comment by arunoda** [comment]
I've got your recommendation and apply it to the README.

File: README.md (hunk 1 above)
Comment (TobyEalden) [inline README.md]: Out of curiosity - why not? Doesn't seem to break the pattern and offers a drop-in replacement for v1 users...

**Pull request closed without merging**